networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/subnets/SyncCommitteeSubscriptionManager.java

```diff
@@ -59,7 +60,7 @@ public synchronized void subscribe(final Integer committeeSubnet, final UInt64 u
           "Update subscription of committee subnet {}, unsubscribe due at slot {}",
           committeeSubnet,
           unsubscribeSlot.toString());
-      subcommitteeToUnsubscribeSlot.put(committeeSubnet, unsubscribeSlot);
+      subcommitteeToUnsubscribeSlot.replace(committeeSubnet, unsubscribeSlot);
```

Comment:
I think we want to stick with `put` here. While we do know there is an existing value so `replace` will wind up doing the same thing, we definitely want to put the new value, so no point in the additional contains check replace will do.

Reply:
Ah good catch. I will revert this change. Sorry I forgot to mention it the PR note.